src/equations/compressible_euler_2d.jl

```diff
@@ -1082,6 +1194,48 @@ end
     return SVector(f1m, f2m, f3m, f4m)
 end
 
+@inline function splitting_lax_friedrichs(u, ::Val{:plus},
+                                          normal_direction::AbstractVector,
+                                          equations::CompressibleEulerEquations2D)
+    rho_e = last(u)
+    rho, v1, v2, p = cons2prim(u, equations)
+
+    a = sqrt(equations.gamma * p / rho)
+    H = (rho_e + p) / rho
+    lambda = 0.5 * (sqrt(v1^2 + v2^2) + a)
+
+    v_normal = v1 * normal_direction[1] + v2 * normal_direction[2]
+    rho_v_normal = rho * v_normal
+
+    f1p = 0.5 * rho_v_normal + lambda * u[1]
+    f2p = 0.5 * rho_v_normal * v1 + 0.5 * p * normal_direction[1] + lambda * u[2]
+    f3p = 0.5 * rho_v_normal * v2 + 0.5 * p * normal_direction[2] + lambda * u[3]
+    f4p = 0.5 * rho_v_normal * H + lambda * u[4]
```

Comment:
This splitting is of the form `normal_direction[1] * splitting_in_direction_1 + normal_direction[2] * splitting_in_direction_2`, isn't it?

Reply:
Not quite, this is only the case on the central flux part. The penalty added that is `± lambda u` does not get hit with any metric terms, which it shouldn't. Especially not in the first and last conservative variable entry because the density and total energy are direction independent.

Reply:
👍